packages/app-store/routing-forms/components/RoutingNavBar.tsx

```diff
@@ -6,7 +6,7 @@ export default function RoutingNavBar({
   form,
   appUrl,
 }: {
-  form: ReturnType<typeof getSerializableForm>;
+  form: Awaited<ReturnType<typeof getSerializableForm>>;
```

Comment:
I still need to verify why it wasn't throwing TypeScript errors earlier because getSerializableForm has always returned promise.